README.md

```diff
@@ -4,13 +4,70 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
-`gem 'webmock-fixtures`
+`gem "webmock-fixtures"`
+
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "rspec"
+require "webmock/fixtures"
+require "webmock/rspec"
+
+# Register our fixture for www.example.org
+WebMock::Fixtures::Manager.register_fixture(
+  :get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+RSpec.describe do
+  describe "a web request to www.example.org" do
+    let!(:manager) { WebMock::Fixtures::Manager.run([:get_example]) }
```

Comment:
I forget, do we need to tell people to set up a `WebMock` teardown in their RSpec config?